src/main/java/frc/robot/subsystems/swerve/SwerveCommands.java

```diff
@@ -45,8 +48,16 @@ public static Command getOpenLoopFieldRelativeDriveCommandSlow(
             DoubleSupplier xSupplier, DoubleSupplier ySupplier, DoubleSupplier thetaSupplier
     ) {
         return new InitExecuteCommand(
-                () -> SWERVE.initializeDrive(false, DriveMode.SLOW),
-                () -> SWERVE.fieldRelativeDrive(xSupplier.getAsDouble(), ySupplier.getAsDouble(), thetaSupplier.getAsDouble()),
+                () -> SWERVE.initializeDrive(new SwerveState(DriveSpeed.SLOW)),
+                () -> SWERVE.drive(xSupplier.getAsDouble(), ySupplier.getAsDouble(), thetaSupplier.getAsDouble()),
+                SWERVE
+        );
+    }
+
+    public static Command getFR_N_O_MOR(DoubleSupplier xSupplier, DoubleSupplier ySupplier, Rotation2d targetAngle) {
```

Comment:
what is this?

Reply:
It's kinda obvious, like read the name. getFieldRelative_Normal_Open_MiddleOfRobot()